office/libreoffice/Portfile

```diff
@@ -144,9 +144,14 @@ depends_run-append \
     port:python${python_version} \
     port:py${python_version}-lxml
 
-if {${os.major} < 14} {
+if {${os.platform} eq "darwin" && ${os.major} < 19} {
+    known_fail yes
     pre-fetch {
-        ui_error "${name} @${version} requires OS X 10.10 or later."
+        ui_error "Building ${name} @${version} requires macOS 10.15 or later."
+        ui_msg ""
+        ui_msg "Consider downloading a version from LibreOffice's website:"
+        ui_msg "https://www.libreoffice.org/download/download/"
+        ui_msg ""
```

Comment:
are the `ui_msg` seen when running `port` at a normal non debug output level ? I suspect note, so better to make them port notes.